cfn.yaml

```diff
@@ -124,11 +124,11 @@ Resources:
     Properties:
         Description: Stage for salesforce message handler API
         RestApiId: !Ref MessageHandlerApi
-        DeploymentId: !Ref MessageHandlerAPIDeployment
+        DeploymentId: !Ref MessageHandlerAPIDeployment1
         StageName: !Sub ${Stage}
     DependsOn: ContactMethod
 
-  MessageHandlerAPIDeployment:
+  MessageHandlerAPIDeployment1:
```

Comment:
This update triggers a redeploy of the API Gateway. Without this change it would've been necessary to manually redeploy via the console.